rustup-init.sh

```diff
@@ -294,19 +305,17 @@ ensure_loongarch_uapi() {
             return 0
             ;;
         234)
-            echo >&2
-            echo 'Your Linux kernel does not provide the ABI required by this Rust' >&2
-            echo 'distribution.  Please check with your OS provider for how to obtain a' >&2
-            echo 'compatible Rust package for your system.' >&2
-            echo >&2
+            err 'Your Linux kernel does not provide the ABI required by this Rust'
+            err 'distribution.  Please check with your OS provider for how to obtain a'
+            err 'compatible Rust package for your system.'
             exit 1
             ;;
         *)
-            echo "warn: Cannot determine current system's ABI flavor, continuing anyway." >&2
-            echo >&2
-            echo 'Note that the official Rust distribution only works with the upstream' >&2
-            echo 'kernel ABI.  Installation will fail if your running kernel happens to be' >&2
-            echo 'incompatible.' >&2
+            warn "Cannot determine current system's ABI flavor, continuing anyway."
+            warn
+            warn 'Note that the official Rust distribution only works with the upstream'
+            warn 'kernel ABI.  Installation will fail if your running kernel happens to be'
+            warn 'incompatible.'
```

Comment:
So ideally those two sections should be reformatted to one sentence per line as well.

Reply:
no problem 👍